lib/apply.js

```diff
@@ -1,4 +1,27 @@
 import map from "../data/map";
+import { scaleWidth, scaleHeight } from "./responsive"
+
+/**
+ * Detect using Switch Case for responsive width, height, text
+ */
+const detectResponsive = style => {
+  switch(style[0]) {
+    case "w":
+      return {
+        width: scaleWidth(style[1])
+      }
+
+    case "h":
+      return {
+        height: scaleHeight(style[1])
+      }
+
+    case "text":
+      return {
+        fontSize: scaleWidth(style[1])
+      }
+  }
+}
```

Comment:
Please use array destructuring assignment so other contributor know what the meaning for index 0 / 1.For example, instead of using `style [0] / style [1]` you can use `const [type, value] = style` or directly on params `fx([type, value])`.